### ISSUE TITLE
Investigate issue with error message offset for users without convergent vision

### ISSUE THREAD

**Issue by hannalaakso** [alphagov/govuk-frontend#1768]
## Background

Issue first discussed [on the backlog](https://github.com/alphagov/govuk-design-system-backlog/issues/47#issuecomment-591360096). 

## Summary of issue

The main finding was that the participant in research doesn’t have convergent vision so they found inputting the second password difficult since the field was offset to the right (with the error styling applied).

This is what they saw when the page reloaded after they input the wrong password:

<img width="756" alt="Screenshot 2020-03-24 at 13 33 11" src="https://user-images.githubusercontent.com/5007934/77431857-33a2f280-6dd5-11ea-97e0-31d2be4666d1.png">

### Markup used in prototype 
I’ve had a look at the markup used for the fields in the prototype which all looks correct. I can share the link/credentials for the prototype.

## My transcript of research audio
_(Shared with consent of research participant)_ 

[Researcher asks participant to enter (made up) passwords in a form]

**Participant:** 
Oh so it wasn’t the same password. For some reason this is difficult for me. 

Another reason to use password managers is that it’s quite hard for me to do that [enter password manually].  

I’m good I’m there. 

For some reason I mistyped the password and what it does is it offsets just because of the way that the UI works, it offsets the password boxes and puts it in a different colour which for me doesn’t really help. Most of what we have on any of these when you’re using green is [inaudible] for me anyway. This has actually highlighted it in a form of orange which is fine but it offsets it from the password boxes. So what that means to me is that I’ve got to use convergent vision which I don’t have. So it took me three attempts to get that right, just because I can’t align my vision with two things one of which is offset, it’s just very hard for me. 

**Researcher:** 
That’s interesting thank you. If you click back we’ll be able to see that issue.

**Participant:** 
Yeah I was wondering about that. So that’s _in red with green_* and I know the green’s generally used but that as a visual cue doesn’t help me in fact confuses. So we tend to use greyscale if we can’t use colour but we would use a colour blind palette for it and we’ve changed all of our UI to do that now since the accessibility directive. But can you see how it’s slightly offset? So I navigate between this password and this password, it’s actually to the right which means that my vision can’t make that adjustment easily so it took me three attempts because the password box is actually offset to the right.

—
_*Spoke to the researcher: they think that this comment referred to the fact that the red for used for error messages is not a “pure” red so could contain hints of colours . However it’s worth noting that the prototype used the legacy red (#b10e1e). As we don’t rely on colour to communicate things and the colour the participant saw isn’t the current one not going to take any further action on this for now. Although their comment about using grayscale colours is interesting!_

## Done when

- [x] We've discussed as a team how to approach the issue
- [x] We've created a follow up issue, if appropriate

**Comment by hannalaakso** [comment]
As discussed with team at triage, we haven't seen this issue being reported before so we're going to see if it comes up again (possibly in the [backlog issue](https://github.com/alphagov/govuk-design-system-backlog/issues/47)) before taking further action.
